pkg/soc/validator_test.go

```diff
@@ -40,7 +40,7 @@ func TestSocValidator(t *testing.T) {
 	}
 
 	// check valid chunk
-	v := soc.NewValidator()
+	v := soc.NewSocValidator()
```

Comment:
Why is this in this PR? And all the other validator naming changes?

Reply:
https://github.com/ethersphere/bee/pull/461/commits/c00182cd1114c466dd7d9421191827257a1f64f3

these come from the rebase conflict fixing commit, will revert this